controlplane/route.cpp

```diff
@@ -966,7 +997,17 @@ void route_t::prefix_flush_prefixes(common::idp::updateGlobalBase::request& glob
 
 	for (auto& [vrf, priority_current_update] : prefixes)
 	{
-		(void)vrf; ///< @todo: VRF
+		uint32_t vrf_id;
```

Comment:
It's better to avoid uninitialized variables.
smth like `uint32_t vrf_id{};` or `uint32_t vrf_id = 0;`
But feel free to ignore